packages/babel-core/test/errors-stacks.js

```diff
@@ -305,3 +306,39 @@ describe("@babel/core errors", function () {
     ).toContain("%done%");
   });
 });
+
+expect.extend({
```

Comment:
I had this matcher in another project and I just copy-pasted it. It allows matching a string to a template literal with regexps in holes. I will eventually extract it to its own little library so that I don't have to copy improvements in two places.